src/components/product/ProductsList.component.tsx

```diff
@@ -0,0 +1,12 @@
+import { products } from '@/api/products.ts';
+import { ProductCardComponent } from '@/components/product/ProductCard.component.tsx';
+
+import styles from './product.module.css';
+
+export const ProductsListComponent = () => (
+    <div className={styles.productsList}>
```

Comment:
You need to add search bar layout: `Added layout of Search Bar`

![Screenshot 2024-05-21 at 19 40 08](https://github.com/3apy6a/fe-react-2024/assets/115080771/33f6dc65-381d-457f-b3c2-e91aae9581f6)